app/controllers/api/api_controller.rb

```diff
@@ -88,7 +89,15 @@ def pept2taxa
   # param[extra]: "true" or "false", Include lineage
   # param[names]: "true" or "false", Include the lineage names
   def pept2lca
-    @result = {}
+    @result = pept2lca_helper
+
+    filter_input_order
+
+    respond_with(@result)
+  end
+
+  def pept2lca_helper
```

Comment:
move to private section

_[Original comment](https://github.ugent.be/unipept/unipept/pull/658#discussion_r16986) by @bmesuere on Thu Oct 11 2018 at 10:13._